src/test/java/org/junit/tests/experimental/theories/runner/UnsuccessfulWithDataPointFields.java

```diff
@@ -121,7 +121,7 @@ public void numbers(int x) {
     public void dataPointFieldsMustBeStatic() {
         assertThat(
                 testResult(DataPointFieldsMustBeStatic.class),
-                CoreMatchers.<PrintableResult>both(failureCountIs(2))
+                CoreMatchers.<PrintableResult>both(failureCountIs(1))
```

Comment:
I've applied this to some other tests where it made sense. See d5fc61c.
Please take a look @stefanbirkner 
Thanks